src/ed.cpp

```diff
@@ -595,7 +597,7 @@ int main(int argc, char** argv)
 
         if (config.hasError())
         {
-            std::cout << std::endl << "Error during configuration:" << std::endl << std::endl << config.error() << std::endl;
+            ROS_ERROR_STREAM(std::endl << "Error during configuration:" << std::endl << std::endl << config.error());
```

Comment:
Why does this one get special treatment?

Reply:
This is in main.

We don't want to use ROS logging in the `singalHandler`, because the program is dying. So we don't know the status of the ROS logging. You don't want more errors covering the original error.